Makefile

```diff
@@ -85,17 +86,17 @@ RUN = $(CONTAINER_CLI) run -t -i --sig-proxy=true -u $(UID):docker --rm \
 	-v /etc/passwd:/etc/passwd:ro \
 	$(DOCKER_SOCKET_MOUNT) \
 	$(CONTAINER_OPTIONS) \
-	--mount type=bind,source="$(PWD)",destination="/work" \
+	--mount type=bind,source="$(PWD)",destination=$(WORK_DIR) \
 	--mount type=volume,source=go,destination="/go" \
 	--mount type=volume,source=gocache,destination="/gocache" \
-	-w /work $(IMG)
+	-w $(WORK_DIR) $(IMG)
 else
 $(info Building with your local toolchain.)
 RUN =
 export GOBIN ?= $(GOPATH)/bin
 endif
 
-MAKE = $(RUN) make --no-print-directory -e -f Makefile.core.mk
+MAKE = $(RUN) make $(MFLAGS) --no-print-directory -e -f Makefile.core.mk
```

Comment:
Where does this come from?

Reply:
It's a Makefile variable which represents the arguments passed on the command line